jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudformation/annotations/ErrorAnnotationTest.kt

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudformation.annotations
+
+import com.intellij.lang.annotation.HighlightSeverity
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+
+class ErrorAnnotationTest {
+
+    @Test
+    fun parsesLinterResponse() {
+        val linterOutput = "[\n" +
+            "  {\n" +
+            "    \"Filename\": \"linterInput.json\",\n" +
+            "    \"Level\": \"Error\",\n" +
+            "    \"Location\": {\n" +
+            "      \"End\": {\n" +
+            "        \"ColumnNumber\": 13,\n" +
+            "        \"LineNumber\": 4\n" +
+            "      },\n" +
+            "      \"Path\": [\n" +
+            "        \"Resources\",\n" +
+            "        \"S3Bucket\",\n" +
+            "        \"Type\"\n" +
+            "      ],\n" +
+            "      \"Start\": {\n" +
+            "        \"ColumnNumber\": 7,\n" +
+            "        \"LineNumber\": 4\n" +
+            "      }\n" +
+            "    },\n" +
+            "    \"Message\": \"Invalid or unsupported Type undefined for resource S3Bucket in us-east-1\",\n" +
+            "    \"Rule\": {\n" +
+            "      \"Description\": \"Making sure the basic CloudFormation resources are properly configured\",\n" +
+            "      \"Id\": \"E3001\",\n" +
+            "      \"ShortDescription\": \"Basic CloudFormation Resource Check\",\n" +
+            "      \"Source\": \"https://github.com/aws-cloudformation/cfn-python-lint\"\n" +
+            "    }\n" +
+            "  }\n" +
+            "]\n"
```

Comment:
https://realkotlin.com/tutorials/2018-06-26-multiline-string-literals-in-kotlin/

(I think I'm warming up to Kotlin)

Reply:
TIL!